src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
The last child of an element should not have a margin to the right to follow the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). You could use the `:not(:last-child)` pseudo-class to apply the margin-right only to the elements that are not the last child.